documentation/autoinstall-quickstart-s390x.md

```diff
@@ -6,15 +6,15 @@ This page is just a slightly adapted page of [the autoinstall quickstart page](a
 
 ## Download an ISO
 
-At the time of writing (just before focal release), the best place to go is here:
-<http://cdimage.ubuntu.com/ubuntu/releases/20.04/release/>
+At the time of writing (just after the kinetic release), the best place to go is here:
+<https://cdimage.ubuntu.com/ubuntu/releases/22.10/release/>
```

Comment:
Certainly good to update this bit.
What do you think about generalizing this mention and changing it to point to https://releases.ubuntu.com/
If we do leave this, there is the mild inconsistency of sometimes pointing to cdimage and sometimes releases, we should pick one (releases would be my preference).

Reply:
Thanks!
Sadly, releases.ubuntu.com only offers downloads for amd64 whereas cdimage.ubuntu.com offers downloads for everything **but** amd64...

There is a link on releases.ubuntu.com:

>The cdimage server also hosts releases of other Ubuntu images not found on this server, such as builds for less popular architectures and other non-standard and unsupported images. For Ubuntu Desktop and Server on popular architectures, please see the links above instead.
>
>[Ports, Unsupported, and Experimental Images for Ubuntu](http://cdimage.ubuntu.com/releases/)

Reply:
oh right